lib/Sema/DerivedConformanceAdditiveArithmetic.cpp

```diff
@@ -54,38 +54,230 @@ static StringRef getMathOperatorName(MathOperator op) {
   llvm_unreachable("invalid math operator kind");
 }
 
+static ValueDecl *getFatalErrorFn(ASTContext &C) {
+  auto fatalErrorLookupRes = llvm::SmallVector<ValueDecl*, 1>{};
+  C.lookupInSwiftModule("fatalError", fatalErrorLookupRes);
```

Comment:
It would be nice to put this in `KnownIdentifiers.def`. Then, you can access it via `C.Id_fatalError`.